src/PowerShellEditorServices.Protocol/LanguageServer/GetCommandsRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.EditorServices.Protocol.LanguageServer
+{
+    public class GetCommandsRequest
+    {
+        public static readonly
+            RequestType<List<PSCommandMessage>, object, object, object> Type =
+            RequestType<List<PSCommandMessage>, object, object, object>.Create("powerShell/getCommands");
+    }
+
+    public class PSCommandMessage
```

Comment:
All the public bits should ideally have comments here. The important part is that either `GetCommandsRequest` or `PSCommandMessage` documents what feature this message services and why it exists. But this is a public API, so we should aim to document it well.